Changelog.md

```diff
@@ -2,6 +2,79 @@
+
+1. Fix weird indentation in `Link.hh`
+    * [Pull request #1324](https://github.com/ignitionrobotics/ign-gazebo/pull/1324)
+
+1. fix buoyancy test by increasing tol
+    * [Pull request #fix buoyancy test by increasing tol](https://github.com/ignitionrobotics/ign-gazebo/pull/fix buoyancy test by increasing tol)
+
+1. Limit thruster system's input thrust cmd
+    * [Pull request #1318](https://github.com/ignitionrobotics/ign-gazebo/pull/1318)
+
+1. Load and run visual plugin (system) on GUI side
+    * [Pull request #1275](https://github.com/ignitionrobotics/ign-gazebo/pull/1275)
+
+1. Log an error if JointPositionController cannot find the joint. (citadel retarget)
+    * [Pull request #1314](https://github.com/ignitionrobotics/ign-gazebo/pull/1314)
+
+1. Update source install instructions
+    * [Pull request #1311](https://github.com/ignitionrobotics/ign-gazebo/pull/1311)
+
+1. Document the `<topic>` option for JointPositionController.
+    * [Pull request #1309](https://github.com/ignitionrobotics/ign-gazebo/pull/1309)
+
+1. Fix typo in EntityComponentManager
+    * [Pull request #1304](https://github.com/ignitionrobotics/ign-gazebo/pull/1304)
+
+1. Buoyancy: fix center of volume's reference frame
+    * [Pull request #1302](https://github.com/ignitionrobotics/ign-gazebo/pull/1302)
+
+1. Fix graded buoyancy problems
+    * [Pull request #1297](https://github.com/ignitionrobotics/ign-gazebo/pull/1297)
+
+1. Add surface to buoyancy engine. (retarget fortress)
+    * [Pull request #1298](https://github.com/ignitionrobotics/ign-gazebo/pull/1298)
+
+1. Remove EachNew calls from sensor PreUpdates
+    * [Pull request #1281](https://github.com/ignitionrobotics/ign-gazebo/pull/1281)
+
+1. Prevent GzScene3D 💥 if another scene is already loaded
+    * [Pull request #1294](https://github.com/ignitionrobotics/ign-gazebo/pull/1294)
+
+1. Fix various typos on API documentation
+    * [Pull request #1291](https://github.com/ignitionrobotics/ign-gazebo/pull/1291)
+
+1. Optional orientation when spawning entity using spherical coordinates
+    * [Pull request #1263](https://github.com/ignitionrobotics/ign-gazebo/pull/1263)
+
+1. Cleanup update call for non-rendering sensors
+    * [Pull request #1282](https://github.com/ignitionrobotics/ign-gazebo/pull/1282)
+
+1. Documentation Error
+    * [Pull request #1285](https://github.com/ignitionrobotics/ign-gazebo/pull/1285)
+
+1. Min and max parameters for velocity, acceleration, and jerk apply to linear and angular separately.
+    * [Pull request #1229](https://github.com/ignitionrobotics/ign-gazebo/pull/1229)
+
+1. Add project() call to examples
+    * [Pull request #1274](https://github.com/ignitionrobotics/ign-gazebo/pull/1274)
+
+1. Implement /server_control::stop
+    * [Pull request #1240](https://github.com/ignitionrobotics/ign-gazebo/pull/1240)
+
+1. 👩‍🌾 Make depth camera tests more robust (#897)
+    * [Pull request #897) (#1257](https://github.com/ignitionrobotics/ign-gazebo/pull/897) (#1257)
```

Comment:
this looks like a backport to citadel than forward ported. I think this should be released already and we can remove this entry

Reply:
[fcb02cb](https://github.com/ignitionrobotics/ign-gazebo/pull/1338/commits/fcb02cb56f8e50e743c70f03fcf43a52008de0d9)